ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.8-presubmits.yaml

```diff
@@ -1312,6 +1312,337 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
```

Comment:
'optional: true` is missing for the jobs

Reply:
Updated:
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1329
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1399
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1471
* https://github.com/openshift/release/pull/15802/files#diff-38238cb6f4ec8238b5e94af12df0ddac3ca1d17cb88e5a1d80c00c0a467c927fR1543
*